sdk/servicebus/service-bus/samples/typescript/src/receiveMessagesStreaming.ts

```diff
@@ -30,26 +36,49 @@ export async function main() {
   // - See session.ts for how to receive using sessions.
   const receiver = sbClient.createReceiver(queueName);
 
-  const processMessage = async (brokeredMessage) => {
-    console.log(`Received message: ${brokeredMessage.body}`);
-    await brokeredMessage.complete();
-  };
-  const processError = async (args) => {
-    console.log(`Error from error source ${args.errorSource} occurred: `, args.error);
-  };
-
   try {
-    receiver.subscribe(
+    const subscription = receiver.subscribe(
       {
-        processMessage,
-        processError
+        processMessage: async (brokeredMessage: ServiceBusReceivedMessage) => {
+          console.log(`Received message: ${brokeredMessage.body}`);
+          await receiver.completeMessage(brokeredMessage);
+        },
+        processError: async (args: ProcessErrorArgs) => {
+          console.log(`Error from source ${args.errorSource} occurred: `, args.error);
```

Comment:
@JoshLove-msft We should be in sync here with what you recommend for the error handler part of the processor in your library.

Reply:
Added as part of #12117